mail/smtp_sender.go

```diff
@@ -10,31 +10,51 @@ import (
 
 //SMTPSender allows to send Emails by connecting to a SMTP server.
 type SMTPSender struct {
-	Dialer *gomail.Dialer
+	Dialer  *gomail.Dialer
+	message *gomail.Message
```

Comment:
No sure this is a good idea to have the message here. How do you handle two concurrent `Send`?